packages/env/src/constants/defaults.ts

```diff
@@ -123,6 +123,11 @@ export const DEFAULTS = {
 
 	FILE_METADATA_ALLOW_LIST: 'ifd0.Make,ifd0.Model,exif.FNumber,exif.ExposureTime,exif.FocalLength,exif.ISOSpeedRatings',
 
+	TUS_ENABLED: false,
+	TUS_CHUNK_SIZE: '10mb',
+	TUS_UPLOAD_EXPIRATION: 600_000, // 10 minutes
```

Comment:
Shall we use an `ms` string for consistency with TTL envs?

```suggestion
	TUS_UPLOAD_EXPIRATION: '10m',
```